src/test/java/org/dependencytrack/resources/v1/VulnerabilityPolicyResourceTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * This file is part of Dependency-Track.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) Steve Springett. All Rights Reserved.
+ */
+
+package org.dependencytrack.resources.v1;
+
+import alpine.server.filters.ApiFilter;
+import alpine.server.filters.AuthenticationFilter;
+import org.apache.http.HttpStatus;
+import org.dependencytrack.model.VulnerabilityPolicy;
+import org.dependencytrack.policy.vulnerability.VulnerabilityPolicyAnalysis;
+import org.dependencytrack.policy.vulnerability.VulnerabilityPolicyRating;
+import org.glassfish.jersey.server.ResourceConfig;
+import org.glassfish.jersey.servlet.ServletContainer;
+import org.glassfish.jersey.test.DeploymentContext;
+import org.glassfish.jersey.test.ServletDeploymentContext;
+import org.junit.Test;
+
+import javax.json.JsonArray;
+import javax.ws.rs.core.Response;
+import java.util.Date;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class VulnerabilityPolicyResourceTest extends AbstractPostgresResourceTest {
+
+    @Override
+    protected DeploymentContext configureDeployment() {
+        return ServletDeploymentContext.forServlet(new ServletContainer(
+                new ResourceConfig(VulnerabilityPolicyResource.class)
+                        .register(ApiFilter.class)
+                        .register(AuthenticationFilter.class)))
+                .build();
+    }
+
```

Comment:
Can we have another test here that checks whether all policy fields are returned as expected? i.e. using `assertThatJson` like so: https://github.com/DependencyTrack/hyades-apiserver/blob/850a98862dade23fb10fc6db6657f520b5019a48/src/test/java/org/dependencytrack/resources/v1/VexResourceTest.java#L127-L204

Reply:
Added a test